pkg/pillar/cmd/zedrouter/networkinstance.go

```diff
@@ -565,6 +565,12 @@ func doNetworkInstanceSanityCheck(
 	log.Infof("Sanity Checking NetworkInstance(%s-%s): type:%d, IpType:%d\n",
 		status.DisplayName, status.UUID, status.Type, status.IpType)
 
+	err := checkNIphysicalPort(ctx, status)
```

Comment:
Why are we not setting the Error in status here?

Reply:
return err here, the handleNetworkInstanceCreate() is going to set this.